lib/ansible/modules/windows/win_get_url.py

```diff
@@ -105,6 +111,10 @@
     type: int
     default: 10
     version_added : '2.4'
+notes:
+- If your URL includes an escaped slash character (%2F) this module will convert it to a real slash.
+  This is a result of the behaviour of the System.Uri class
+  L(as described in the documentation,https://docs.microsoft.com/en-us/dotnet/framework/configure-apps/file-schema/network/schemesettings-element-uri-settings#remarks).
```

Comment:
Is there documentation on this, I tried to build the doc locally with these changes and it just linked to itself and shows the URL as text. This also fails on validation due to the line length.

Reply:
It is documented here: https://docs.ansible.com/ansible/devel/dev_guide/developing_modules_documenting.html#linking-within-module-documentation

It seems I am missing the space after the comma, and maybe we can split it over 2 lines ?

Reply:
When reading that it seems like `L()` is to link to internal documentation and not external links? I could be wrong here I've just never used it before.

Reply:
You think so because of the example, but grep for 'L(' and you'll see it works for external links too. We have introduced this for a number of places in the ACI documentation where having a very long link in the documentation really looks awful and is quite unhelpful to the cause.

Reply:
It is just silly that the long-line test is also extended to documentation, where to be honest, limiting lines is a nuisance. You really would like to have one paragraph be one long line so you don't have to redo the whole paragraph line per line because you changed the wording somewhere.